packages/multichain-network-controller/package.json

```diff
@@ -68,7 +68,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^25.0.0",
+    "@metamask/accounts-controller": "^26.0.0",
```

Comment:
Since we are updating its peer, we should release this controller too!

Let's use the `0.2.0` for this one.